Makefile

```diff
@@ -1,6 +1,7 @@
 # Minimal makefile to build Antora documentation
 BUILDDIR = docs
 PLAYBOOK = antora-playbook.yml 
+LOCAL_PLAYBOOK = antora-playbook-local.yml 
```

Comment:
This filename differs from that referenced in `make preview`?

Reply:
there are now 3 targets,  docs: makes docs using clone of the repo, 'preview': makes sdp-docs locally, clones the other repos, 'local-preview': which only makes the sdp-docs locally, so I don't have to keep dealing with the zscaler repo clone failure

Reply:
added and pushed the 'antora-playbook-local-only.yml'  file